src/interpreter/internal.rs

```diff
@@ -155,6 +155,20 @@ impl<S> Interpreter<S> {
         Ok(())
     }
 
+    /// Reduces the unspent balance of the asset defined in $fp
+    pub(crate) fn external_asset_balance_sub(&mut self, value: Word) -> Result<(), RuntimeError> {
+        let fp = self.registers[REG_FP] as usize;
+        let asset_id = &self.memory[fp..fp + AssetId::LEN];
```

Comment:
Isn't the `asset_id` the second set of 32 bytes on the [call frame](https://github.com/FuelLabs/fuel-specs/blob/master/specs/vm/main.md#call-frames)?

Reply:
There should be a test for this is what this means!

Reply:
This function was added as a specific use-case for SMO and is no longer present